actionpack/lib/action_controller/metal/helpers.rb

```diff
@@ -49,7 +54,11 @@ module ActionController
   #   N/A | Carolina Railhawks Training Workshop
   #
   module Helpers
-    extend ActiveSupport::Concern
+    extend ActiveSupport::      # Extract helper names from files in <tt>app/helpers/**/*_helper.rb</tt>
+      def all_application_helpers
+        all_helpers_from_path(helpers_path)
+      end
+    endConcern
```

Comment:
This looks accidental, could you undo this change?

Reply:
Yes, this one is a strange one: without the addition of those spaces, (2) spaces, the build fails: due to rubocop. 

I can revert the change, but the build will not pass. Please confirm.

Reply:
There are several new lines added here, and they're not valid Ruby syntax. It looks like a copy and paste error.

Don't worry about placating RuboCop if its complaints are unrelated to your changes.

Reply:
Ok I see what you mean. That's very strange. I have no idea how they came into this commit. I'll revert it.

Reply:
ok i think we have a winner here! thx for the reviewing, i surely appreciate it.